src/components/App/App.js

```diff
@@ -11,7 +12,9 @@ function App() {
       <Section type='benefits' layout='four-columns' title='Benefits'>
```

Comment:
Should this Section tag be removed to avoid conflict with my changes to the benefits section? This was a placeholder to show off Card and CardList.

Reply:
Could you sort how you want to resolve since you added both? Is there a reason you aren't just updating the placeholder in your PR? If it's alright by you, I think we should only add placeholders if we're going to work off of them; otherwise we're inviting a lot of confusion and back and forth

Reply:
Yeah, Iyou're right. I'm not sure why I didn't just work off of the placeholder. I am very much not used to working with other people